test/simplifyExpression/multiplyFractionsSearch/multiplyFractionsSearch.test.js

```diff
@@ -8,9 +8,13 @@ function testMultiplyFractionsSearch(exprString, outputStr) {
 
 describe('multiplyFractions', function () {
```

Comment:
hm it's sort of odd now that multiplyFractions also cancels fractions. you should probably rename multiplyFractions

... or you could call multiply fractions *in* the cancelling code so cancelling always happens in `cancelLikeTerms` instead of sometimes happening in `multiplyFractions` - what do you think of that? it seems a bit more intuitive to me